syft/cataloger/python/parse_setup.go

```diff
@@ -0,0 +1,46 @@
+package python
+
+import (
+	"bufio"
+	"io"
+	"regexp"
+	"strings"
+
+	"github.com/anchore/syft/syft/cataloger/common"
+	"github.com/anchore/syft/syft/pkg"
+)
+
+// integrity check
+var _ common.ParserFn = parseSetup
+
+var pinnedDependency = regexp.MustCompile(`['"]\W?(\w+\W?==\W?[\w\.]*)`)
```

Comment:
nit: would be useful to add a few example matches in comments to illustrate to devs what this is trying to match on (similar to https://github.com/anchore/syft/blob/ac9141e2ada35bcbc07f29f8942d894956ea4b28/syft/cataloger/java/archive_filename.go#L13-L16)

Reply:
Great suggestion, I'll add those

Reply:
This is now improved!